howto/anbox/harden.md

```diff
@@ -0,0 +1,71 @@
+(howto-harden)=
+# How to harden your deployment
+
+Anbox Cloud is designed with secure development practices but there are certain aspects to consider on your side to make sure the entire deployment is secure.
+
+## General guidelines
+
+Consider the following simple yet impactful measures to ensure that you run a secure Anbox Cloud deployment:
+
+- Always run the latest supported version of Anbox Cloud and keep your deployment up to date. See {ref}`ref-release-notes`.
+- Make sure your deployment uses machines running a supported Ubuntu version. See {ref}`ref-requirements`.
+- Do not disable TLS pinning when you are not using a load balancer.
+- If you find a security issue, report it to the [Ubuntu security team](https://wiki.ubuntu.com/SecurityTeam/FAQ#Contact).
+
+## Reduce attack surface
+
+The following diagram illustrates the attack surface and potential points of threat.
+
+![Anbox Cloud attack surface|690x398](/images/anbox_attack_surface.svg)
+
+### 1 - Web client
+
+- Make sure you have `CIS/USG` and Livepatch enabled on Ubuntu Pro. See {ref}`enable Livepatch <https://documentation.ubuntu.com/pro/pro-client/enable_livepatch/>` and {ref}`enable CIS or USG <https://documentation.ubuntu.com/pro/pro-client/enable_cis/>` for more information.
```

Comment:
Only enabling CIS or USG is not enough, you also have to apply both up to a profile which satisfies the users requirements. Maybe we should put it more in the way of that people should consider both to further harden their systems?